.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
What is this file ?

Reply:
This is the project file of the phpstorm IDE. Would you like me to remove it?

Reply:
I don't think we need to remove it. The .gitignore contains a bunch of stuff like this already anyway.

Reply:
NotI disagree. You can easily add a global gitignore for your needs. See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Cool I learned something new today! I've now removed that line from the .gitignore file